proofs/tree.go

```diff
@@ -201,12 +201,12 @@ type TreeOptions struct {
 	CompactProperties bool
 }
 
-type Salt struct {
+type SaltPair struct {
```

Comment:
Can't we move this to example/... or tests? I don't think it's used here except for tests, right?

Reply:
You are right, then can be deleted safely